index.html

```diff
@@ -3,11 +3,62 @@
 <head>
     <title>Academics | NUS Computing</title>
     <link rel="stylesheet" href="assets/css/bootstrap.css">
+    <link rel="stylesheet" href="assets/css/stylesheet.css">
     <link rel="shortcut icon" type="image/x-icon" href="assets/img/favicon.ico">
     <script src="assets/js/bootstrap.js"></script>
+    <!-- to allow dropdown of menu on mobile -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 </head>
 <body>
```

Comment:
As mentioned, wrap the body contents in a <div class="container">. check http://getbootstrap.com/css/#overview-container for more info